### PR TITLE
luizfds/c02-network07

### PR DESCRIPTION
**Pull request by LuizFDS**
# Description

Exercise: C02-Network07
Student Name: Luiz Silva

Added file with an explanation about what I did to solve this exercise.

## Type of change

- [x] Exercise Submission
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/02class/exercises/c02-network07/luizfds/c02-network07.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    A NAT Gateway does something similar, but with two main differences:
+        1. It allows resources in a private subnet to access the internet (think yum updates, external database connections, wget calls, OS patch, etc)
+        2. It only works one way. The internet at large cannot get through your NAT to your private resources unless you explicitly allow it.
+
+- Can you delete the destination route to your VPC network? Why?
+    No, by default, each VPC comes with 1 route table pre-configured with a “local” route. The scope of the “local” route is only within the subnet defined for the entire VPC. For example, if your VPC was set up to have the address space of 172.16.0.0/16, your “local” route would be defined as “172.16.0.0/16”. This allows all of the resources created within the VPC to talk to each other without any additional configuration. You cannot delete the “local” route from your route table, and anytime a new route table is created within a VPC, the “local” route is included by default.
+
+- What happens if you do not associate your route table with any subnets?
+    Your route table will not be used.
```
Comment (caiocezart) [inline classes/02class/exercises/c02-network07/luizfds/c02-network07.txt]: then which one will be used instead?

Comment (LuizFDS) [reply classes/02class/exercises/c02-network07/luizfds/c02-network07.txt]: The Main route table will be used instead. Each subnet in our VPC must be associated with a route table and a subnet can be explicitly associated with custom route table, or implicitly or explicitly associated with the main route table. It's not possible to delete the main route table, but you can replace the main route table with a custom subnet route table that you've created.

Comment (caiocezart) [reply classes/02class/exercises/c02-network07/luizfds/c02-network07.txt]: nice! can you add this to your answer?

Comment (LuizFDS) [reply classes/02class/exercises/c02-network07/luizfds/c02-network07.txt]: It's already in there.

**Pull request merged**